packages/protocol/src/interaction/EventDataDecoder.ts

```diff
@@ -121,6 +138,40 @@ export function normalizeAndDecodeEventData(
     return result;
 }
 
+export function normalizeEventStatus(data: TypeFromSchema<typeof TlvEventStatus>[]): DecodedEventReportStatus[] {
```

Comment:
This is another candidate for iterable input/generator/removal of typeof

Reply:
added TODO